README.md

```diff
@@ -1,9 +1,87 @@
+Community Team. We use this repo to track meeting agendas, new ideas, and concerns
+that cut across our various [subteams](#subteams).
 
-The team meets on Wednesdays at 5PM UTC on the the mozilla.org irc network in the #rust-community channel. If you don't have an irc client you can access it via [Mibbit](http://chat.mibbit.com/?server=irc.mozilla.org&channel=%23rust-community). The meeting agendas can be found [here](https://docs.google.com/document/d/1ncFvzgb85ay5lNECV6myw4uOv5rjsvtyoLcgyQTJZ9E/edit?usp=sharing).
+## Get Involved!
+
+### 💕 Participate in a Subteam
+
+Working with or joining a Subteam is the best way to get involved with the Rust
+Community team. Subteams are subject focused and bring together people who are
+invested in managing and producing awesome events, content, workshops and more!
+
+See more on subteams [below](#subteams).
+
+### 🏢 Attend a General Meeting
+
+The Rust Community Team meets every other Wednesday, 5PM UTC on the the mozilla.org
+IRC network in the #rust-community channel. These meetings are treated as a standup
+for the subteams, though we occasionally host guests. Before you consider joining a
+general meeting- take a look at the [Subteams](#subteams) to get a sense for what
+sort of work is being done!
+
+If you don't have an irc client you can access it via [Mibbit](http://chat.mibbit.com/?server=irc.mozilla.org&channel=%23rust-community). The meeting agendas can be found [here](https://docs.google.com/document/d/1ncFvzgb85ay5lNECV6myw4uOv5rjsvtyoLcgyQTJZ9E/edit?usp=sharing).
+
+If you would like to attend, please read our [Observer Policy](/rfcs/observer-policy.md).
+
+## Subteams
+
+The subteams are the best way to get involved in the Community Team. You can learn
+more about each team by visiting it's repository, reading it's goal document, or 
+messaging it's Team Lead. You can find all the details below- we're excited to hear from
+you!
+
+### 🗓️ Events
+
+The events team helps events to get better in touch with the rest of the community. We advise and bring you in contact with other organisers. The events team are also the ones to approach if you want to get you meetup featured on the events calendar and subsequently the newsletter.
+
+We are hoping to find new members around the globe, especially outside of Europe and the US.
+
+[**Repo**](https://github.com/rust-community/events-team/)
+ | [**Goals Document**](https://docs.google.com/document/d/1jH2Cz493ILQ79mTR1O8Msgf4v7UmaYp5Mc0UjTNmQ68/edit#)
+ | **Leader**: Florian Gilcher ([@skade])
+
+### 📚 Content
+
+Organize and write for the community blog and other publication outlets, Organise and promote material like videos and podcasts, Provide lectorate (copy editing) and assistance for people that want to write blog posts on rust
+
+[**Repo**](https://github.com/rust-community/content-team) 
+ | [**Goals Document**](https://docs.google.com/document/d/1fNKaOiV-D2VHdNUPJoGFsM2D10RWNsTGj5WjGHLboqg)
```

Comment:
Just added our goals to the repo, could you please reference this instead? https://github.com/rust-community/content-team/blob/master/GOALS.md